webapp/channels/src/sass/components/_mentions.scss

```diff
@@ -21,6 +21,15 @@ del .group-mention-link:focus {
     color: variables.$white;
 }
 
+.mention-link {
+    color:rgb(56, 111, 229)
```

Comment:
question, which color is this one? where did you get it from? can you please use one of the colors we have as a variable value for supporting the theming?